test/user_test.rb

```diff
@@ -0,0 +1,37 @@
+require 'minitest/autorun'
+require 'minitest/pride'
+require './lib/joke'
+require './lib/user'
+
+class UserTest < Minitest::Test
+  def setup
+    @sal = User.new("Sal")
+    @ali = User.new("Ali")
+    @joke_1 = Joke.new(1, "Why did the strawberry cross the road?", "Because his mother was in a jam.")
+    @joke_2 = Joke.new(2, "How do you keep a lion from charging?", "Take away its credit cards.")
+  end
+
+  def test_it_exists
+    assert_instance_of User, @sal
+  end
+
+  def test_it_has_attributes
+    assert_equal "Sal", @sal.name
+    assert_equal "Ali", @ali.name
+    assert_equal [], @sal.jokes
```

Comment:
good work in here. you don't really need to test Ali here, because you've tested the functionality for one User object already, so if it works for one, it'll work for others